src/compiler/sys.ts

```diff
@@ -36,23 +36,6 @@ namespace ts {
         Low = 250
     }
 
-    function getPriorityValues(highPriorityValue: number): [number, number, number] {
-        const mediumPriorityValue = highPriorityValue * 2;
-        const lowPriorityValue = mediumPriorityValue * 4;
-        return [highPriorityValue, mediumPriorityValue, lowPriorityValue];
-    }
-
-    function pollingInterval(watchPriority: PollingInterval): number {
-        return pollingIntervalsForPriority[watchPriority];

```

Comment:
The line above causes an error with this PR because we now correctly include `undefined` in the type because `watchPriority` is one of the values 250, 500, or 2000, all of which are out of range of the tuple type. This code is basically bogus, but also never used.